File: eval/baselines/checkout/jetpack-compose/CheckoutScreen.kt
// BASELINE — intentionally-naive "no-skill" reference for the eval harness (do NOT
// copy). What an assistant typically emits WITHOUT the Mobile UI/UX Pro Max skill:
// hardcoded colors, off-grid spacing, sub-legible price text, and only the happy
// path — none of the required UI states, so nothing guards a double-charge.
// Graded against examples/checkout/jetpack-compose/.
package baseline.checkout

import androidx.compose.foundation.layout.*
import androidx.compose.material3.*
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp

@Composable
fun CheckoutScreen() {
    Column(modifier = Modifier.padding(18.dp)) {
        Text("Order summary", color = Color(0xFF111827), fontSize = 18.sp)
        Spacer(Modifier.height(14.dp))
        Row {
            Text("Wireless Headphones", color = Color(0xFF111827))
            Spacer(Modifier.weight(1f))
            Text("$129.00", fontSize = 10.sp)
        }
        Text("Tax and shipping calculated at charge", color = Color(0xFF9CA3AF), fontSize = 10.sp)
        Spacer(Modifier.height(22.dp))
        Button(
            onClick = { },
            colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF16A34A)),
            modifier = Modifier.fillMaxWidth().padding(15.dp),
        ) {
            Text("Pay now", color = Color(0xFFFFFFFF))
        }
    }
}

File: eval/baselines/checkout/swiftui/CheckoutScreen.swift
// BASELINE — intentionally-naive "no-skill" reference for the eval harness (do NOT
// copy). What an assistant typically emits WITHOUT the Mobile UI/UX Pro Max skill:
// hardcoded hex colors, off-grid spacing, a fixed-height price row, an undersized
// stepper, and only the happy path — none of the required UI states, so nothing
// guards a double-charge. Graded against examples/checkout/swiftui/. (Assumes a
// Color(hex:) extension, as such code usually does.)
import SwiftUI

struct CheckoutScreen: View {
    @State private var qty = 1

    var body: some View {
        VStack(alignment: .leading) {
            Text("Order summary").foregroundColor(Color(hex: "#111827")).frame(height: 26)
            HStack {
                Text("Wireless Headphones")
                Spacer()
                Text("$129.00")
            }
            .padding(15)
            HStack {
                Button(action: { qty -= 1 }) { Text("-") }.frame(width: 32, height: 32)
                Text("\(qty)")
                Button(action: { qty += 1 }) { Text("+") }.frame(width: 32, height: 32)
            }
            Text("Tax and shipping calculated at charge").foregroundColor(Color(hex: "#9CA3AF"))
            Button(action: {}) { Text("Pay now").frame(maxWidth: .infinity) }
                .padding(15)
                .background(Color(hex: "#16A34A"))
        }
        .padding(18)
    }
}
